client/index.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-undef */
+
+// const { default: axios } = require('axios')
+
+/* ---------------------------- */
+// REMEMBER TO FLIP THIS WHEN NOT USING LIVE SERVER
+
+const BASE_URL = 'http://localhost:3001/api'
+// const BASE_URL = '/api'
+
+/* ---------------------------- */
+
+const entriesSection = document.querySelector('.entries-container')
+const infoSection = document.querySelector('h6')
+
+const entriesArray = []
+
+function createEntry(id, name, number) {
+  const entry = document.createElement('div')
+  const deleteBtn = document.createElement('button')
+  entry.className = 'entry'
```

Comment:
It might be worth producing a function that accepts arguments (class, text, tag name) to create an element instead of doing it twice.